frontend/src/components/Collections/components/DeleteCollection/index.tsx

```diff
@@ -37,14 +37,7 @@ const DeleteCollection: FC<Props> = ({ id }) => {
 
   return (
     <>
-      <Button
-        intent={Intent.DANGER}
-        minimal
-        text="Delete"
-        icon={IconNames.TRASH}
-        onClick={toggleAlert}
-        onMouseEnter={handleHover}
-      />
+      <Button onClick={toggleAlert} onMouseEnter={handleHover} />
```

Comment:
`<DeleteCollection />` gets `Button` from `props`, which is a styled Blueprint `<MenuItem />`, so the only relevant props we need to pass on are `onClick` and `onMouseEnter`